src/components/mainDoctor/emptyWrapper/emptyWrapper.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { EmptyWrapper, EmptyIcon, EmptyText } from '../styles'
+
+const EmptyState = () => {
+  return (
+    <EmptyWrapper>
```

Comment:
But ints not empty, it has content in it.

Reply:
I will rename this component for better understanding